app/models/story.rb

```diff
@@ -95,4 +105,12 @@ def sets
   def has_media?
     ore_aggregation.edm_web_resources.any?(&:media?)
   end
+
+  def guardian_consent?
```

Comment:
`#guardian_consent?` and `#age_confirm?` are automatically added to the model by Mongoid and need not be declared explicitly.